mkdocs.yml

```diff
@@ -1,3 +1,4 @@
+---
```

Comment:
We prefer to use 2 spaces wherever possible as a convention, so could you please remove this change.

Reply:
Sure. I suggest an editorcinfig file for the repository since two spaces is a variant from the common four spaces for YAML files. 

http://editorconfig.org

Changes coming up.